pyomo/contrib/trustregion/TRF.py

```diff
@@ -1,45 +1,41 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
 import numpy as np
 from math import pow
 from numpy import inf
 from numpy.linalg import norm
 from pyomo.contrib.trustregion.param import *
-from pyomo.contrib.trustregion.filterMethod import FilterElement, Filter
-from pyomo.contrib.trustregion.helper import cloneXYZ, packXYZ, minIgnoreNone, maxIgnoreNone
-from pyomo.contrib.trustregion.Logger import IterLog, Logger
-from pyomo.contrib.trustregion.PyomoInterface import PyomoInterface, ROMType
-
-
+from pyomo.contrib.trustregion.filterMethod import (
+    FilterElement, Filter)
+from pyomo.contrib.trustregion.helper import (cloneXYZ, packXYZ,
+    minIgnoreNone, maxIgnoreNone)
+from pyomo.contrib.trustregion.Logger import (IterLog, Logger)
+from pyomo.contrib.trustregion.PyomoInterface import (
+    PyomoInterface, ROMType)
 
 def TRF(m,eflist):
+    """The main function of the Trust Region Filter algorithm
+    m is a PyomoModel containing ExternalFunction() objects Model
```

Comment:
There should be a blank line between the 1-line "short description" and the remainder of the docstring